doc/rst/language/evolution.rst

```diff
@@ -27,28 +52,72 @@ their scope is now a user error:
     begin {
       A += 1;
     }
-    // Error: A destroyed here at function end, but the begin could still be using it!
+    // Error: A destroyed here at function end, but the begin could still
+    // be using it!
   }
 
 
+
 Similarly, using a slice after an array has been destroyed is an error:
 
 .. code-block:: chapel
 
   proc badBeginSlice() {
     var A: [1..10000] int;
-    var slice => A[1..1000];
+    // slice1 sets up a slice using the => operator
+    // note that the => operator is deprecated (see below)
+    var slice1 => A[1..1000];
+    // slice2 sets up a slice by creating a reference to it
+    ref slice2 A[1..1000];
+    // either way, using the slice in a begin that can continue
+    // after the function declaring theh array exits is an error
     begin {
-      slice += 1;
+      slice1 += 1;
+      slice2 += 1;
     }
-    // Error: A destroyed here at function end, but the begin tries to use it through the slice!
+    // Error: A destroyed here at function end, but the begin tries to
+    // use it through the slices!
   }
 
+.. _readme-evolution.alias-operator-deprecated:
+
+array alias operator deprecated
+*******************************
+
+The array alias operator, `=>`, has been deprecated in Chapel 1.15.
+Previously, the supported way to declare one array that aliases another
+(or a slice of another) was to use `=>`. Now, the supported way to do
+that is to use a `ref` or `const ref` variable:
+
+For example, before Chapel 1.15 you might have written:
+
+.. code-block:: chapel
+
+  // pre-1.15
+  var A:[1..10] int;
+  // set up a const alias of A
+  const alias => A;
+  // set up a mutable slice of A
+  var slice => A[2..5];
```

Comment:
You should probably add a third example of using the reindexing-based '=>' form:

````chapel
var A0: [0..9] = A;
````

which would now be written:

````chapel
ref A0 = A.reindex({0..9});
````